app/views/components/_email-signup-section.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  details ||= {}
+  link_text ||= false
+  link_href ||= false
+  intro ||= false
+  full_width ||= false
+  track_action ||= false
+  track_label ||= false
+  transparent_background ||= false
+  css_classes ||= false
+  classes ||= []
+  tracking_data = track_action ? {
```

Comment:
We generally avoid adding tracking to components by default, I think this is better added per instance.

Reply:
Changed to passing all tracking data from outside the component.